phoenix-core/src/it/java/org/apache/phoenix/end2end/DateTimeIT.java

```diff
@@ -57,12 +44,13 @@
 import org.junit.Test;
 
 
-public class DateTimeIT extends BaseHBaseManagedTimeIT {
+public class DateTimeIT extends BaseHBaseManagedTimeTableReuseIT {
 
     protected Connection conn;
     protected Date date;
     protected static final String tenantId = getOrganizationId();
     protected final static String ROW10 = "00D123122312312";
+    protected  String tableName;
```

Comment:
Don't have tableName as a member variable. Instead each test case can create their own table. This will be critical once we decide on parallelizing test methods within a test class.